elements/tags.lua

```diff
@@ -857,6 +891,11 @@ local function Untag(self, fs)
 	self.__tags[fs] = nil
 end
 
+local function strip(tag)
+	-- remove prefix, output length, custom args, and suffix
+	return tag:gsub('%[.-%$>', '['):gsub('{.-}%]', ']'):gsub('%(.-%)%]', ']'):gsub('$<.-%]', ']')
```

Comment:
You are now [officially on par](https://github.com/oUF-wow/oUF_MovableFrames/blob/731a86b021c1132826606501204e7a0b2a54170c/movable.lua#L310-L315) with @haste 🥇